apollo-router/src/configuration/metrics.rs

```diff
@@ -206,7 +206,7 @@ impl Metrics {
         );
         log_usage_metrics!(
             value.apollo.router.config.persisted_queries,
-            "$.preview_persisted_queries[?(@.enabled == true)]",
+            "$.persisted_queries[?(@.enabled == true)]",
```

Comment:
@BrynCooke I just want to double-check my understanding that this expression is only ever applied to a YAML file that has been "migrated".

Reply:
That's correct.